<ui version="4.0" >
 <class>Service</class>
 <widget class="QWidget" name="Service" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Service</string>
  </property>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTabWidget" name="serviceBox" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>WEB</string>
       </attribute>
       <widget class="QWidget" name="verticalLayout_2" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QTabWidget" name="tabWidget_2" >
           <property name="currentIndex" >
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_3" >
            <attribute name="title" >
             <string>Pre-offer</string>
            </attribute>
            <widget class="QWidget" name="verticalLayout_3" >
             <property name="geometry" >
              <rect>
               <x>9</x>
               <y>9</y>
               <width>231</width>
               <height>161</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_8" >
                  <property name="text" >
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_5" >
                  <property name="maxLength" >
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="offerWebList" />
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_12" >
                  <property name="text" >
                   <string>Domain</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_6" >
                  <property name="maxLength" >
                   <number>100</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_4" >
            <attribute name="title" >
             <string>More</string>
            </attribute>
            <widget class="QGroupBox" name="groupBox_2" >
             <property name="geometry" >
              <rect>
               <x>11</x>
               <y>9</y>
               <width>231</width>
               <height>151</height>
              </rect>
             </property>
             <property name="title" >
              <string>Web Settings</string>
             </property>
             <widget class="QWidget" name="verticalLayout_4" >
              <property name="geometry" >
               <rect>
                <x>10</x>
                <y>20</y>
                <width>211</width>
                <height>110</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" >
               <item>
                <layout class="QGridLayout" >
                 <item row="1" column="0" >
                  <widget class="QLabel" name="label_3" >
                   <property name="text" >
                    <string>Space</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0" >
                  <widget class="QLabel" name="label_4" >
                   <property name="text" >
                    <string>Number of database</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0" >
                  <widget class="QLabel" name="label_6" >
                   <property name="text" >
                    <string>Domain</string>
                   </property>
                   <property name="buddy" >
                    <cstring>repassword</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1" >
                  <widget class="QLineEdit" name="repassword" >
                   <property name="maxLength" >
                    <number>100</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1" >
                  <widget class="QComboBox" name="comboBox" >
                   <item>
                    <property name="text" >
                     <string>100</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>150</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>200</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>250</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>300</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>350</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>400</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>450</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>500</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>550</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>600</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>650</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>700</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>750</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>800</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>850</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>900</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>950</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>1000</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="2" column="1" >
                  <widget class="QComboBox" name="comboBox_2" >
                   <item>
                    <property name="text" >
                     <string>1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>2</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>3</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>4</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>5</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>6</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>7</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>8</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>9</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>10</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="0" column="1" >
                  <widget class="QLineEdit" name="lineEdit" >
                   <property name="maxLength" >
                    <number>20</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0" >
                  <widget class="QLabel" name="label" >
                   <property name="text" >
                    <string>Name</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>STREAM</string>
       </attribute>
       <widget class="QWidget" name="verticalLayout_7" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QTabWidget" name="tabWidget_3" >
           <property name="currentIndex" >
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_7" >
            <attribute name="title" >
             <string>Pre-offer</string>
            </attribute>
            <widget class="QWidget" name="verticalLayout_8" >
             <property name="geometry" >
              <rect>
               <x>9</x>
               <y>9</y>
               <width>231</width>
               <height>160</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_5" >
                  <property name="text" >
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_3" >
                  <property name="maxLength" >
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="offerStreamList" />
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_7" >
                  <property name="text" >
                   <string>Title</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_4" />
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_8" >
            <attribute name="title" >
             <string>More</string>
            </attribute>
            <widget class="QGroupBox" name="groupBox_4" >
             <property name="geometry" >
              <rect>
               <x>11</x>
               <y>9</y>
               <width>231</width>
               <height>151</height>
              </rect>
             </property>
             <property name="title" >
              <string>Stream Settings</string>
             </property>
             <widget class="QWidget" name="verticalLayout_9" >
              <property name="geometry" >
               <rect>
                <x>10</x>
                <y>20</y>
                <width>211</width>
                <height>110</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" >
               <item>
                <layout class="QGridLayout" >
                 <item row="1" column="0" >
                  <widget class="QLabel" name="label_9" >
                   <property name="text" >
                    <string>Slots</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0" >
                  <widget class="QLabel" name="label_10" >
                   <property name="text" >
                    <string>Bits</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0" >
                  <widget class="QLabel" name="label_11" >
                   <property name="text" >
                    <string>Title</string>
                   </property>
                   <property name="buddy" >
                    <cstring>repassword</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1" >
                  <widget class="QLineEdit" name="repassword_3" >
                   <property name="maxLength" >
                    <number>20</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1" >
                  <widget class="QComboBox" name="comboBox_5" >
                   <item>
                    <property name="text" >
                     <string>5</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>10</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>15</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>20</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>25</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>30</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>35</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>40</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>45</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>50</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>55</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>60</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>65</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>70</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>75</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>80</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>85</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>90</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>95</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>100</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>200</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>300</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>400</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>500</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="2" column="1" >
                  <widget class="QComboBox" name="comboBox_6" >
                   <item>
                    <property name="text" >
                     <string>24</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>32</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>48</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>64</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>128</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>256</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="0" column="1" >
                  <widget class="QLineEdit" name="lineEdit_2" >
                   <property name="maxLength" >
                    <number>20</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0" >
                  <widget class="QLabel" name="label_2" >
                   <property name="text" >
                    <string>Name</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton" >
        <property name="text" >
         <string>More</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOk_3" >
        <property name="text" >
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel_3" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serviceBox</tabstop>
  <tabstop>tabWidget_2</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>offerWebList</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButtonOk_3</tabstop>
  <tabstop>pushButtonCancel_3</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>repassword</tabstop>
  <tabstop>tabWidget_3</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>comboBox_5</tabstop>
  <tabstop>comboBox_6</tabstop>
  <tabstop>repassword_3</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>offerStreamList</tabstop>
  <tabstop>lineEdit_4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>offerWebList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>pushButtonOk_3</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>75</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel" >
     <x>190</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offerStreamList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>pushButtonOk_3</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>141</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel" >
     <x>191</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
